TLM/TLM/U/UIScaler.cs

```diff
@@ -14,7 +13,7 @@ public static class UIScaler {
         /// <summary>Caching because UIView.Instance.uiCamera can be null sometimes.</summary>
         private static float cachedGuiHeight = GUI_HEIGHT;
 
-        /// <summary>Screen width for GUI is always fixed at 1920.</summary>
+        /// <summary>Gets screen width for GUI (always fixed at 1920).</summary>
```

Comment:
what if there is different aspect ratio? 
see https://github.com/kianzarrin/Skylines-ModTools/blob/master/Debugger/UI/UIScaler.cs
There is some guard code because the resolution is not ready at startup.

Reply:
This code exists for several releases, i remember i've done some testing and found that UI canvas size and aspect stays 1920x1080 at all times. If that's not true, i will need more info.